src/Rcl/Web/Templates.hs

```diff
@@ -1,20 +1,21 @@
+{-# LANGUAGE TypeApplications #-}
```

Comment:
Could update `package.yaml` with our preferred `default-extensions`. I don't know that that was a thing when I made this package.